_posts/2021-04-27-Optimising-ux-for-software.markdown

```diff
@@ -7,70 +7,65 @@ author: Lucy Sharp
+There is value in designers being part of a VSM workshop. They have greater context around how the current system operates, the constraints, gaps and opportunities and how that is feeding into poor user experience. For example, with a recent client looking at AI/ML, we did not have complete understanding of how the AI/ML worked, which meant that when we created new screens we made assumptions about the flow. It was not until we re-engaged the client that we had more context. This lack of knowledge was a constraint; it slowed us down. Now VSM is not integral to DevOps, however, VSM provides visibility into the constraints and gaps within a system and this visibility is integral to the value of DevOps. Mapping value from idea to customer provides a designer with more context on the current system and an idea of future state. This context coupled with user research leads to data driven decisions supporting delivery.
 <br /><br />
 
-## Why UX and DevOps
+## UX and DevOps ROI
 
-DevOps without UX is possible however your return on investment and delivery success is impeded and we see this with website latency costing $2.6 billion ([Design Resources ](https://techjury.net/blog/user-experience-stats/#gref)). I would not recommend it. It is all well and good to accelerate technology delivery but that value is lost if there is no clear strategy and no understanding of users.
+DevOps without UX is unfortantely common and ROI is impeded. Understanding users ensures that the flow of value from DevOps delivers a competitive advantage.
 
-Understanding users ensures that the flow of value from DevOps delivers a competitive advantage. Performance issues contribute to 90% of users leaving an app and functionality and design issues cause 86% of users to delete or uninstall an app ([Ewelina Luszczek](https://medium.com/@ewelina.luszczek/what-is-the-business-value-of-ux-design-affc3a1e370b)). This is not a successful deployment. Instead, when UX works together with DevOps, the inherent value of UX is unlocked providing clearer strategy and direction to DevOps. This is the competitive advantage.
+Performance issues contribute to 90% of users leaving an app; functionality and design issues cause 86% of users to delete or uninstall an app ([Ewelina Luszczek](https://medium.com/@ewelina.luszczek/what-is-the-business-value-of-ux-design-affc3a1e370b)). Not to mention that website latency costs $2.6 billion ([Design Resources ](https://techjury.net/blog/user-experience-stats/#gref)). This is not a successful delivery. Instead, when UX works together with DevOps, the inherent value of UX is unlocked providing clearer strategy and direction to DevOps. This is the competitive advantage.
```

Comment:
This flows better now (my previous comments were prior to this commit).

The 2.6bn stat still seems dubious though - and you should reference the original article (or close to)